sklearn/datasets/base.py

```diff
@@ -263,6 +269,7 @@ def load_iris():
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
         full description of the dataset.
+    (data,target) : tuple if return_X_y == True
```

Comment:
sorry, i mean that there should be a newline above the new return type but not one below it.